app/code/Magebit/Faq1/Model/Question.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magebit\Faq1\Model;
+
+use Magento\Framework\Model\AbstractModel;
+use Magebit\Faq1\Model\ResourceModel\Question as ResourceModel;
+use Magebit\Faq1\Api\Data\QuestionInterface;
+
+/**
+ * Model class extends AbstractModel and implements QuestionInterface. Question interface will force model class to define getId, getQuestion,
+ * etc... methods which will return a unique id. _init method in constuctor will define the resource model that actually fetches information from database.
+ */
+class Question extends AbstractModel implements QuestionInterface
```

Comment:
Update Interace with parameter and return types, this will also requre an update to match the interface file